packages/frameworks/src/frameworks.ts

```diff
@@ -2006,6 +2006,51 @@ export const frameworks = [
       },
     ],
   },
+  {
+    name: 'Sanity',
+    slug: 'sanity',
+    demo: 'https://www.sanity.io/get-started',
```

Comment:
This should point to a demo deployment, but don't worry about that for now - we will take care of it.

Reply:
For the record: The proposal is to add a Sanity Studio to `/examples` that can be deployed through Vercel's project creation and require the Sanity Integration. Then it will mint a project and deploy with the correct environment variables that will make the deployed studio work.